Finjinon/PhotoCaptureViewController.swift

```diff
@@ -31,7 +31,7 @@ public protocol PhotoCaptureViewControllerDelegate: NSObjectProtocol {
 
 open class PhotoCaptureViewController: UIViewController, PhotoCollectionViewLayoutDelegate {
     open weak var delegate: PhotoCaptureViewControllerDelegate?
-    open var imagePickerAdapter: ImagePickerAdapter = ImagePickerControllerAdapter()
+    open var imagePickerAdapter: ImagePickerAdapter? = ImagePickerControllerAdapter()
```

Comment:
This should probably have some documentation.
As it is now optional, I also think it would make sense _not_ to instantiate it by default.